src/hptlink.c

```diff
@@ -772,8 +772,13 @@ void linkArea(s_area * area)
 void usage(void)
 {
     printf(
-        "Usage: hptlink [options] [areaname ...]\n" "Options:  -t\t- build reply TREE\n" "\t  -s\t- do not use Subject\n" "\t  -a\t- search in all messages (for singlethread only)\n" "\t  -r\t- do not use REPLY:/MSGID:\n"
-                                                                                                                                                                                       "\t  -n\t- link with 'new' messages only ('new' from last linked + 1)\n");
+        "Usage: hptlink [-c config] [options] [areaname ...]\n"
+        "Options:\n"
+        "\t  -t\t- build reply TREE\n"
+        "\t  -s\t- do not use Subject\n"
+        "\t  -a\t- search in all messages (for singlethread only)\n"
+        "\t  -r\t- do not use REPLY:/MSGID:\n"
+        "\t  -n\t- link with 'new' messages only ('new' from last linked + 1)\n");
```

Comment:
"\t  -n\t- link with the 'new' messages only ('new' from the last linked + 1)\n");